lib/reverse_string.rb

```diff
@@ -1,5 +1,16 @@
 def reverse_characters(str)
+    #create an array dividing up the string into individual characters
+    array = str.split("")
+    #create a holder for to-be-reversed characters
+    reverse_array = []
+    #going through each character in the array, store it in the last position of the new holder array
+    array.each do |n|
+        reverse_array.insert[-1,n]
```

Comment:
In your [prompt response](https://github.com/bethanyhaubert/ruby-prompt__arrays/blob/master/lib/arrays.md#how-do-you-add-elements-to-an-array), you wrote:

```
prime.insert(-1, 53)
 inserts 53 in the -1 position of the array
prime << 53
 puts 53 at the last positon of the array
prime.push(53).push(59)
 pushes the new elements to the end of the array. Able to chain multiple pushes together
```

So I notice two things:

1. `insert` has parentheses after it–not square brackets. Your code here, however, uses square brackets.
2. `push` seems to also fit the bill.

Try one or the other–it might not matter–and see how that affects things.